backend_jobs/ingestion_pipeline/main.py

```diff
@@ -0,0 +1,94 @@
+"""
+  Copyright 2020 Google LLC
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ """
+
+from __future__ import absolute_import
+from datetime import datetime
+import argparse
+import logging
+import apache_beam as beam
+from apache_beam.io import WriteToText
+from apache_beam.options.pipeline_options import PipelineOptions
+from providers import image_provider_flickr
+from pipeline_lib import database_functions
+
+#Global name for the job that will run
+JOB_NAME = 'ingestion' + str(int(datetime.timestamp(datetime.now())))
+IMAGE_PROVIDERS = {'FlickrProvider': image_provider_flickr.FlickrProvider}
+
+def filtered_images(element):
+    """
+    This function returns true if the image standes in all of the requirement's. e.g.:; url != none
+    """
+    return (element.url is not None and
+    element.coordinates is not None and
+    element.format is not None and
+    element.resolution['width'] >100 and
```

Comment:
add spacing

Reply:
I am not sure you need 'is not None', maybe just if element.url and element.coordinate and etc.

Reply:
yes you are right thanks